corehq/apps/app_manager/models.py

```diff
@@ -1988,29 +1990,29 @@ class Detail(IndexedSchema, CaseListLookupMixin):
     get_tabs = IndexedSchema.Getter('tabs')
 
     sort_elements = SchemaListProperty(SortElement)
-    filter = StringProperty()
+    filter = StringProperty(exclude_if_none=True)
 
-    instance_name = StringProperty(default='casedb')
+    instance_name = StringProperty(default='casedb', exclude_if_none=True)
```

Comment:
I would expect that properties with defaults would never be None

Reply:
good point, removed `exclude_if_none` for defaults that aren't falsy